src/Microsoft.ComponentDetection.Detectors/go/GoComponentDetector.cs

```diff
@@ -52,10 +53,12 @@ protected override async Task OnFileFound(ProcessRequest processRequest, IDictio
             var wasGoCliScanSuccessful = false;
             try
             {
-                if (IsGoCliManuallyEnabled())
+                if (!IsGoCliManuallyDisabled())
                 {
-                    Logger.LogInfo("Go cli scan was manually enabled");
```

Comment:
We should keep this log, Go CLI was found in system and will be used to generate dependency graph. Detection time can be improved by activating fallback strategy (https://aka.ms/go-detection-fallback-strategy), but it will introduce noise into the detected component list.

Reply:
added the suggested logging statement in `USeGoCliToScan` only after validating that go-cli exist in the environment.